pkg/interop/neogointernal/call.go

```diff
@@ -0,0 +1,12 @@
+package neogointernal
+
+// CallWithToken performs contract call using CALLT instruction. It only works
+// for static script hashes and methods, requiring additional metadata compared to
+// ordinary contract.Call. It's more efficient though.
+func CallWithToken(scriptHash string, method string, flags int, args ...interface{}) interface{} {
```

Comment:
Why can't we expose this API to external users?

Reply:
It's kinda exposed (nothing forbids importing `neogointernal`, it's just that there are no guarantees for it), but
 * it needs strictly constant parameters 
 * this particular definition inherently belongs to `neogointernal` where all intrinsincs reside
 * a public generic wrapper around it can't really work at the moment, because inlining code will create local variables for hash/method/flags

IMO for non-native contracts #2221 is the way to go, autogenerated wrapper can use this intrinsic (but it might be uninlined, hmm).

Reply:
Why it might be unhandled? I think importing `neogointernal` will solve the problem, because in this case we don't change the way how `isSyscall` works:
```
return fun.pkg.Name() == "neogointernal" && (strings.HasPrefix(fun.name, "Syscall") ||
		strings.HasPrefix(fun.name, "Opcode") || strings.HasPrefix(fun.name, "CallWithToken"))
}
```

Reply:
It will, I'm thinking more about outer wrapper, but there will be no difference to native contracts, so it shouldn't worry us.